zookeeper-server/src/main/java/org/apache/zookeeper/common/SSLUtil.java

```diff
@@ -0,0 +1,90 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.zookeeper.common;
+
+import org.apache.zookeeper.common.crypto.Crypt;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class SSLUtil {
+
+    private static final Logger LOG = LoggerFactory.getLogger(SSLUtil.class);
+    private static Crypt crypt;
+
+    /**
+     * This method will return an instance of the user configured Crypto class
+     * value for {@value ZKConfig#CONFIG_CRYPT_CLASS}.
+     *
+     * @return an instance of the Crypto class configured
+     */
+    private static Crypt getCrypt() throws ReflectiveOperationException {
```

Comment:
If you wanted to make this lazy-initialized, I think it would be beneficial to make this method thread-safe. Otherwise you could just do this in the static constructor.